Makefile

```diff
@@ -43,6 +43,7 @@ Makefile.common.$(MAKE_BRANCH):
 	rm -f Makefile.common.*
 	curl --fail $(MAKE_REPO)/Makefile.common -o "$@"
 
+GOFLAGS = -buildvcs=false
```

Comment:
Is this passed in as an environment variable or build flag?

Reply:
As env. It will make the docker-run command add it to go-build's env, which then will be picked up by go.

```
mkdir -p .go-pkg-cache bin /home/rd/go/pkg/mod && docker run --rm --net=host --init -e GOPRIVATE=github.com/tigera/* -v /run/user/1000/keyring/ssh:/ssh-agent --env SSH_AUTH_SOCK=/ssh-agent -e GO111MODULE=on -v /home/rd/go/pkg/mod:/go/pkg/mod:rw -e LOCAL_USER_ID=1000 -e GOCACHE=/go-cache -e GOARCH=amd64 -e GOPATH=/go -e OS=linux -e GOOS=linux -e GOFLAGS=-buildvcs=false -v /home/rd/go/src/github.com/tigera/key-cert-provisioner:/go/src/github.com/tigera/key-cert-provisioner:rw -v /home/rd/go/src/github.com/tigera/key-cert-provisioner/.go-pkg-cache:/go-cache:rw -w /go/src/github.com/tigera/key-cert-provisioner -e CGO_ENABLED=1 -e CGO_LDFLAGS= -e CGO_CFLAGS= calico/go-build:v0.90 sh -c 'git config --global url."ssh://git@github.com/".insteadOf "https://github.com/"; GOEXPERIMENT=boringcrypto go build -o  bin/test-signer-amd64 -tags fipsstrict,osusergo,netgo -v -ldflags " -linkmode external -extldflags -static -s -w"  test-signer/test-signer.go && strings  bin/test-signer-amd64 | grep '_Cfunc__goboringcrypto_' 1> /dev/null'
```